src/app/routes/ChainAccounts.tsx

```diff
@@ -38,7 +38,7 @@ function ChainAccounts() {
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
         waspClientService
             .corecontracts()
-            .accountsGetAccounts({ chainID })
+            .accountsGetAccounts(chainID)
```

Comment:
same here, i cant see the base tokens for a selected account
before: 
![image](https://github.com/iotaledger/wasp-dashboard/assets/3624944/651bf581-2534-4870-ae73-fe183a10465e)

after:
![image](https://github.com/iotaledger/wasp-dashboard/assets/3624944/e972ea7b-60bc-4409-b799-7692070ec42a)